### PR TITLE
EVG-763: validate command inside of functions

### PR DESCRIPTION
**Pull request by Kimchelly**
JIRA: https://jira.mongodb.org/browse/EVG-763

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2019-01-22"
+	ClientVersion = "2019-02-19"
```
Comment (john-m-liu) [inline config.go]: The validator is on the server, so you don't actually need to change the client version

Comment (bsamek) [reply config.go]: Ah, sorry, that's my fault.

File: validator/project_validator_test.go [hunk 2]
```diff
@@ -1032,6 +1032,21 @@ func TestValidatePluginCommands(t *testing.T) {
 			So(validatePluginCommands(project), ShouldNotResemble, ValidationErrors{})
 			So(len(validatePluginCommands(project)), ShouldEqual, 1)
 		})
+		Convey("an error should be thrown if a function plugin command doesn't have commands", func() {
+			project := &model.Project{
+				Functions: map[string]*model.YAMLCommandSet{
+					"funcOne": {
+						SingleCommand: &model.PluginCommandConf{
```
Comment (bsamek) [inline validator/project_validator_test.go]: I'm not sure I follow. Doesn't this function _have_ a command?

Comment (john-m-liu) [reply validator/project_validator_test.go]: There's some ambiguity in terminology - the ticket is referring to the command field (ie. shell.exec) being blank. The data structure here is also called a command which contains the command itself and metadata

Comment (tychoish) [reply validator/project_validator_test.go]: it seems like we only added "is the command list empty" validation, and not "is the command misspecified.?" 

Is it possible to add something that attempts to parse the command? by calling command.Render() on the command? My gut is that this is a cycle, but4 could also be a problem

File: config.go (hunk 1 above)
Comment (Kimchelly) [reply config.go]: I undid this change.

File: validator/project_validator_test.go (hunk 2 above)
Comment (Kimchelly) [reply validator/project_validator_test.go]: The command should already be parsed and the existence of the command should be checked in `validateCommands` when it calls `Render()`: https://github.com/evergreen-ci/evergreen/blob/65e7945eed4d707198982cc71890b7e1e12b43f1/validator/project_validator.go#L708

**Comment by Kimchelly** [comment]
evergreen merge

**Pull request merged**